### DISCUSSION TITLE
How do I access the job of flink, which is created by calling flink API?

### DISCUSSION THREAD

**Discussion by xiangzirui** [apache/skywalking#12125] [Q&A]
I have tried to search for the corresponding solution, and the solution they give is to add it when the flink client submits the data, as shown below.
But the way I created flink job was by calling API. Are there any suggestions that can solve my problem?
`bin/flink run -m yarn-cluster -yD env.java.opts: "-javaagent:/opt/flink-1.13.6/skywalking-agent/skywalking-agent.jar=agent.service_name=540yh::calcu-ontime-task,collector.backend_service=192.168.100.22:30180"  --yarnqueue "test"  yourFlinkApp.jar `

**Comment by wu-sheng** [comment]
Typically, we haven't tested to use the agent in flink client or similar case.

https://skywalking.apache.org/docs/skywalking-java/latest/en/setup/service-agent/java-agent/supported-list/

There is no flink plugin existing.
